<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mybutton</class>
 <widget class="QWidget" name="mybutton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="newbutton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="newbutton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="newbutton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="newbutton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="newbutton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="newbutton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="newbutton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="newbutton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="newbutton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>0</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="newbutton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>newbutton</class>
   <extends>QPushButton</extends>
   <header location="global">newbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
